### PR TITLE
[MOB-3442] update message list item styling

### PR DESCRIPTION
**Pull request by evangreer91**
## 🔹 JIRA Ticket(s) if any

* [MOB-3442] (https://iterable.atlassian.net/browse/MOB-3442)

## ✏️ Description

This pull request updates the styling of the message list item and the delete slider to account for varying heights of the message list item.

[MOB-3442]: https://iterable.atlassian.net/browse/MOB-3442

### CODE REVIEW

File: ts/IterableInboxCustomizations.ts [hunk 1]
```diff
@@ -36,11 +36,11 @@ type IterableInboxCustomizations = {
       paddingBottom: number
    },
  
-   body: {
-      fontSize: number,
-      color: string,
-      paddingBottom: number
-   },
+   // body: {
```
Comment (roninopf) [inline ts/IterableInboxCustomizations.ts:39]: Delete?

File: ts/IterableInboxMessageDisplay.tsx [hunk 2]
```diff
@@ -32,6 +33,18 @@ const IterableInboxMessageDisplay = ({
    const messageTitle = rowViewModel.inAppMessage.inboxMetadata?.title
    const [inAppContent, setInAppContent] = useState<IterableHtmlInAppContent>(new IterableHtmlInAppContent(new IterableEdgeInsets(0, 0, 0, 0), ""))
 
+   const sampleHTML = `
```
Comment (roninopf) [inline ts/IterableInboxMessageDisplay.tsx:36]: Not needed anymore I'm guessing?

File: ts/IterableInboxCustomizations.ts (hunk 1 above)
Comment (evangreer91) [reply ts/IterableInboxCustomizations.ts:39]: This will be deleted and addressed on PR-220.

File: ts/IterableInboxMessageDisplay.tsx (hunk 2 above)
Comment (evangreer91) [reply ts/IterableInboxMessageDisplay.tsx:36]: This will be removed.

**Pull request closed without merging**